spec/models/like_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before(:all) do
+    @user = User.create(
+      name: 'cosmo',
+      photo: 'https://Cosmo.jpg',
+      bio: 'Cosmo bio',
+      posts_counter: 0
+    )
+    @post = Post.create(
+      title: 'A new Post',
+      text: 'Web Developer',
+      author_id: @user.id,
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+
+  it '@users comments_counter should increase by one' do
+    @comment = Comment.create(text: 'Hello World!', author_id: @user.id, post_id: @post.id)
+    @comment = Comment.create(text: 'Welcome World!!', author_id: @user.id, post_id: @post.id)
+    expect(@comment.author_id).to_not eq(30)
+  end
```

Comment:
- [ ] The same here. As requested by the previous reviewer, Kindly add the missing tests for the Like model. You are missing a test to test the `update_likes_counter` method of your Like model, please add this ✍🏻

```ruby
  def update_likes_counter
    post.increment!(:likes_counter)
  end
```
- Note you need to test the method, actually, you're checking the author_id on line (23)